common/reducers/auth.js

```diff
@@ -5,10 +5,13 @@ import {UPDATE_USER_REQUEST, UPDATE_USER_SUCCESS, UPDATE_USER_FAILURE} from 'src
 const initialState = {
   currentUser: null,
   lgJWT: null,
+  isAdmin: false,
   isBusy: false,
 }
 
 export function auth(state = initialState, action) {
+  const currentUser = Object.assign({}, state.currentUser, action.currentUser)
+  const isAdmin = currentUser.roles ? currentUser.roles.includes('admin') : false
```

Comment:
See comment above; we should be using permission-based auth rather than role-based auth. Specify in `userCan` that only admins can `viewUsers`, then you need only check in the container that `userCan(currentUser, 'viewUsers')`.